core/engine/src/builtins/string/mod.rs

```diff
@@ -22,7 +22,10 @@ use crate::{
     value::IntegerOrInfinity,
     Context, JsArgs, JsResult, JsString, JsValue,
 };
-use boa_macros::{js_str, utf16};
+#[cfg(feature = "annex-b")]
```

Comment:
Nit: move this next to line 38-39, which already has a feature. Keep the feature flags together.